docker/Dockerfile

```diff
@@ -6,8 +6,6 @@ ARG NUM_PROCESSES
 ENV NUM_PROCESSES ${NUM_PROCESSES:-4}
 ARG NUM_THREADS
 ENV NUM_THREADS ${NUM_THREADS:-2}
```

Comment:
Is NUM_THREADS used anymore? 

Also we should remove WSGI_CONTAINER (and NUM_THREADS if the answer to above is no) from the Helm chart `values.yaml` file, under `env`.

Reply:
No It is not needed as Num threads are Required only when WSGI_CONTAINER is uwsgi, since we will always use Gunicorn , it should be ok to remove Num_Threads from Dockerfile
Also removed the WSGI_CONTAINER and Num_Threads  from values.yaml and also removed uwsgi from the file setup.py as it is being removed, tested these new code changes and the driver works fine and is handling graceful shutdown

![image](https://user-images.githubusercontent.com/79189065/135219750-45d1cb4f-208a-4ae6-b0ae-00539991a543.png)